src/gui/transferlistwidget.cpp

```diff
@@ -147,12 +147,12 @@ TransferListWidget::TransferListWidget(QWidget *parent, MainWindow *main_window)
 
     // Listen for list events
     connect(this, SIGNAL(doubleClicked(QModelIndex)), this, SLOT(torrentDoubleClicked(QModelIndex)));
-    connect(this, SIGNAL(customContextMenuRequested(const QPoint &)), this, SLOT(displayListMenu(const QPoint &)));
+    connect(this, SIGNAL(customContextMenuRequested(const QPoint&)), this, SLOT(displayListMenu(const QPoint&)));
     header()->setContextMenuPolicy(Qt::CustomContextMenu);
-    connect(header(), SIGNAL(customContextMenuRequested(const QPoint &)), this, SLOT(displayDLHoSMenu(const QPoint &)));
-    connect(header(), SIGNAL(sectionMoved(int, int, int)), this, SLOT(saveSettings()));
-    connect(header(), SIGNAL(sectionResized(int, int, int)), this, SLOT(saveSettings()));
-    connect(header(), SIGNAL(sortIndicatorChanged(int, Qt::SortOrder)), this, SLOT(saveSettings()));
+    connect(header(), SIGNAL(customContextMenuRequested(const QPoint&)), this, SLOT(displayDLHoSMenu(const QPoint&)));
+    connect(header(), SIGNAL(sectionMoved(int,int,int)), this, SLOT(saveSettings()));
```

Comment:
>What you said in the postscript is just the opposite to the definition which is given in documentation for QMetaObject::normalizedSignature(). Google says that the fallback mechanism is used when the signature which came from connect() is not normalised and thus there were no exact match found

Keep in mind, that I didn't say it as an argument to keep the old way. I was just "useless" knowledge that I unloaded and as it turns out I was remembering it in reverse.
So this may actually be an argument to relax the rule.